### PR TITLE
Translate /docs/reference/issues-security/security.md in Korea

### PR DESCRIPTION
**Pull request by reung37**
문서를 번역했습니다. 리뷰 부탁드리겠습니다.
from #17686
/language ko

### CODE REVIEW

File: content/ko/docs/reference/issues-security/security.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### 언제 취약점을 보고하지 말아야 하는가?
+- 보안을 위해 쿠버네티스 구성요소를 조정하는데 도움이 필요한 경우
+- 보안 관련 업데이트를 적용하는 데 도움이 필요한 경우
+- 보안 관련 문제가 아닌 경우
+## 보안 취약점 대응
+각 보고서는 제품 보안 위원회 위원들에 의해 작업일 3일 이내에 승인 및 분석한다. [보안 릴리스 프로세스](https://git.k8s.io/security/security-release-process.md#disclosures)가 시작된다.
+제품 보안 위원회와 공유하는 모든 취약성 정보는 쿠버네티스 프로젝트 내에 있으며, 문제를 해결할 필요가 없는 한 다른 프로젝트에 전파되지 않는다.
+보안 문제가 3가지 문제에서 확인된 해결책으로 이동함에 따라, 우리는 리포터를 계속 업데이트할 것이다.
+## 공개 시기
+공개 날짜는 쿠버네티스 제품 보안 위원회와 버그 제출자가 협상합니다. 사용자 완화가 가능해지면 가능한 빨리 버그를 완전히 공개하는 것이 좋습니다. 버그 또는 픽스가 아직 완전히 이해되지 않았거나 솔루션이 제대로 테스트되지 않았을 경우에는 공급 업체 조정을 위해 공개를 지연시키는 것이 합리적입니다. 공개 기간은 즉시(특히 이미 공개적으로 알려진 경우)부터 몇 주까지입니다. 간단한 완화 기능이 있는 취약점의 경우 보고 날짜부터 공개 날짜까지는 7일 정도 소요될 것으로 예상됩니다. 쿠버네티스 제품 보안 위원회는 공개 날짜를 설정할 때 최종 결정권을 갖습니다.
```
Comment (ClaudiaJKang) [inline content/ko/docs/reference/issues-security/security.md]: > *origin*
> A public disclosure date is negotiated by the Kubernetes Product Security Committee and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for vendor coordination. The timeframe for disclosure is from immediate (especially if it's already publicly known) to a few weeks. For a vulnerability with a straightforward mitigation, we expect report date to disclosure date to be on the order of 7 days. The Kubernetes Product Security Committee holds the final say when setting a disclosure date.

```suggestion
공개 날짜는 쿠버네티스 제품 보안 위원회와 버그 제출자가 협상한다. 사용자 완화가 가능해지면 가능한 빨리 버그를 완전히 공개하는 것이 좋다. 버그 또는 픽스가 아직 완전히 이해되지 않았거나 솔루션이 제대로 테스트되지 않았거나 벤더 협력을 위해 공개를 지연시키는 것이 합리적이다. 공개 기간은 즉시(특히 이미 공개적으로 알려진 경우)부터 몇 주까지입니다. 간단한 완화 기능이 있는 취약점의 경우 보고 날짜부터 공개 날짜까지는 7일 정도 소요될 것으로 예상된다. 쿠버네티스 제품 보안 위원회는 공개 날짜를 설정할 때 최종 결정권을 갖는다.
```

File: content/ko/docs/reference/issues-security/security.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+[이 링크](https://groups.google.com/forum/feed/kubernetes-announce/msgs/rss_v2_0.xml?num=50)를 사용하기 위해서 RSS 피드를 구독할 수 있다. 
+## 취약점 보고
+우리는 쿠버네티스 오픈소스 커뮤니티에 취약점을 보고하는 보안 연구원들과 사용자들에게 매우 감사하고 있다. 모든 보고서는 한 그룹의 지역 봉사자들에 의해 철저히 조사된다.
+보고서를 작성하기 위해서는 [모든 쿠버네티스 버그 보고서](https://git.k8s.io/kubernetes/.github/ISSUE_TEMPLATE/bug-report.md)에 대해 예상되는 보안 세부 사항과 함께 [security@kubernetes.io](mailto:security@kubernetes.io)로 이메일을 보냅니다.
+[제품 보안 위원회 구성원](https://git.k8s.io/security/security-release-process.md#product-security-committee-psc)의 GPG 키를 사용하여 이 목록으로 이메일을 암호화할 수 있다. GPG를 사용한 암호화는 공개할 필요가 없다.
+### 언제 취약점을 보고해야 하는가?
+- 쿠버네티스에서 잠재적인 보안 취약점을 발견했다고 생각하는 경우
+- 취약성이 쿠버네티스에 어떤 영향을 미치는지 확신할 수 없는 경우
+- 쿠버네티스가 의존하는 다른 프로젝트에서 취약점을 발견한 경우
+  - 자체 취약성 보고 및 공개 프로세스가 있는 프로젝트의 경우 직접 보고하십시오.
```
Comment (ClaudiaJKang) [inline content/ko/docs/reference/issues-security/security.md]:
```suggestion
  - 자체 취약성 보고 및 공개 프로세스가 있는 프로젝트의 경우 직접 보고한다.
```

**Comment by seokho-son** [comment]
/unassign @gochist
/unassign @seokho-son

@reung37 특별히 특정 리뷰어/승인자에게 해당 PR을 할당해야 하는 경우가 아니라면, 직접 assign을 해주실 필요는 없습니다 :) 팀에서 돌아가면서 assign을 수행하는 PR wrangler 가 지정되고 PR wrangler가 해당 업무를 수행하고 있습니다. (이번에는 제가.. ^^ )

**Comment by ClaudiaJKang** [comment]
@reung37 리뷰 반영 감사합니다! 작업 브랜치가 변경되어, `dev-1.16-ko.6` 기준 리베이스 작업이 필요합니다.

**Comment by gochist** [comment]
duplicated with #17763 
/close

**Pull request closed without merging**